content/en/docs/tutorials/stateless-application/expose-external-ip-address.md

```diff
@@ -12,7 +12,7 @@ external IP address.
 ## {{% heading "prerequisites" %}}
 
 * Install [kubectl](/docs/tasks/tools/).
-* Use a cloud provider like Google Kubernetes Engine or Amazon Web Services to
+* Use a cloud provider like Google Kubernetes Engine, Azure Kubernetes Services or Amazon Web Services to
```

Comment:
nit: AKS is “Azure Kubernetes Service”.

Reply:
also: we should decide if we're naming cloud providers (AWS, Azure, Google) or specific products (EKS, AKS, GKE), and be consistent between all the providers we list.

I'd prefer to list providers not products. This tutorial is applicable if you deploy vanilla Kubernetes to those providers, provided that you set up a suitable cloud-controller-manager integration with your chosen infrastructure platform. It's _also_ applicable if you use managed Kubernetes (AKS, EKS, GKE, etc).

Reply:
alright, so are we going to use (AWS, Azure, Google) in naming cloud provider instead of specific products?

Reply:
What's important to me is that we pick one approach and stick to it.

Reply:
I also think listing cloud providers is appropriate here.
```suggestion
* Use a cloud provider like Google Cloud Platform, Microsoft Azure, or Amazon Web Service to
```

Reply:
@llauderesv are you willing to make this change to provide a list of cloud providers (Google Cloud Platform, Microsoft Azure, or Amazon Web Services) instead of products?

Reply:
@a-mccarthy I think it would be better if we specify directly the list of products that cloud providers should offer.

Reply:
Hi @llauderesv

This task requires that you have a load balancer from a cloud provider. It does not require managed Kubernetes (you can also deploy open source Kubernetes onto servers, hosted at that cloud provider, that you manage yourself). For that reason, we should not list managed Kubernetes services as a prerequisite.

It _is_ OK to document that the cluster you deploy need to have an integration with that cloud provider set up.